sdk/identity/Azure.Identity/tests/ManagedIdentityCredentialArcLiveTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Security.KeyVault.Secrets;
+using NUnit.Framework;
+
+namespace Azure.Identity.Tests
+{
+    public class ManagedIdentityCredentialArcLiveTests : ManagedIdentityCredentialLiveTestBase
+    {
+        public ManagedIdentityCredentialArcLiveTests(bool isAsync) : base(isAsync)
+        {
+            Matcher = new ArcMiRecordedTestMatcher();
+        }
+
+        [NonParallelizable]
+        [Test]
+        public async Task ValidateSystemAssignedIdentity()
+        {
+            if (string.IsNullOrEmpty(TestEnvironment.ArcEnable))
+            {
+                Assert.Ignore();
+            }
+
+            using (ReadOrRestoreManagedIdentityEnvironment())
+            {
+                var vaultUri = new Uri(TestEnvironment.SystemAssignedVault);
+
+                var cred = new ManagedIdentityCredential(options: InstrumentClientOptions(new TokenCredentialOptions()));
```

Comment:
Don't we need to call InstumentClient for  cred here?

Reply:
I've instrumented the `SecretClient` which calls into this credential, both cannot be instrumented since the sync methods on `SecretClient` will call into `GetToken` and not `GetTokenAsync`